src/databaseInterface/src/main/java/org/vaccineimpact/api/db/Routines.java

```diff
@@ -69,27 +68,6 @@ public static void enableTrigger(Configuration configuration, String tableName,
         p.execute(configuration);
     }
 
-    /**
```

Comment:
Isn't this file auto-generated? Similarly the one below?

Reply:
Yup. I've changed the xml file that configures how it is generated. See main-database.xml.